src/world/Explosion.php

```diff
@@ -71,6 +81,8 @@ public function __construct(
 			throw new \InvalidArgumentException("Explosion radius must be greater than 0, got $radius");
 		}
 		$this->subChunkExplorer = new SubChunkExplorer($this->world);
+
+		$this->excludedBlockTypeIds = array_flip($excludedBlockTypeIds);
```

Comment:
Prefer `array_fill_keys($keys, true)` instead of doing this, it has less confusing behaviour